lib/flatpak-installation.c

```diff
@@ -174,6 +174,57 @@ flatpak_get_supported_arches (void)
   return (const char * const *)flatpak_get_arches ();
 }
 
+/**
+ * flatpak_get_system_installations:
+ * @cancellable: (nullable): a #GCancellable
+ * @error: return location for a #GError
+ *
+ * Lists the system installations according to the current configuration and current
+ * availability (e.g. doesn't return a configured installation if not reachable).
+ *
+ * Returns: (transfer full) (element-type FlatpakInstallation): an GPtrArray of
+ *   #FlatpakInstallation instances
```

Comment:
Don't you need to mention that the GPtrArray has a free func set here? I mean, how do bindings know that they don't have to specifically free the elements even though its transfer full?